src/components/wallet/WalletWidgetMini/selectors.js

```diff
@@ -23,27 +24,31 @@ export const selectWallet = (blockchain, address) => createSelector(
     if (multisigWallet) {
       const balance: BalanceModel = multisigWallet.balances().item(mainSymbol)
       return {
-        address: multisigWallet.address(),
-        blockchain: multisigWallet.blockchain(),
+        address,
```

Comment:
Common note: there are a lot of data returns from this selector. And it is better to implement a comparator for the selectors result. Also it using too heavy inputs (the reason of memory leak?).

Also it is clearer to rewrite it like this:
```
export const selectWallet = (blockchain, address) => createSelector(
  [
    getMainWallet,
    getMultisigWallets,
  ],
  (mainWallet: MainWalletModel, multisigWallets: MultisigWalletCollection) => {

    const mainSymbol = getMainSymbolForBlockchain(blockchain)
    const multisigWallet = multisigWallets.item(address)

    if (!multisigWallet) {
      return {
        address,
        blockchain,
        name: mainWallet.name(blockchain, address),
        requiredSignatures: null,
        owners: null,
        releaseTime: null,
        pendingCount: null,
        isMultisig: false,
        isTimeLocked: false,
        is2FA: false,
        isDerived: false,
        customTokens: null,
        amount: balance.amount(),
        isMain: true,
      }
    }

    const balance: BalanceModel = multisigWallet.balances().item(mainSymbol)
    return {
      address,
      blockchain,
      name: multisigWallet.name(),
      requiredSignatures: multisigWallet.requiredSignatures && multisigWallet.requiredSignatures(),
      pendingCount: multisigWallet.pendingCount && multisigWallet.pendingCount(),
      isMultisig: multisigWallet.isMultisig(),
      isTimeLocked: multisigWallet.isTimeLocked(),
      owners: multisigWallet.owners ? multisigWallet.owners().items().map((owner: OwnerModel) => owner.address()) : null,
      is2FA: multisigWallet.is2FA(),
      isDerived: multisigWallet.isDerived(),
      customTokens: multisigWallet.customTokens ? multisigWallet.customTokens() : null,
      releaseTime: multisigWallet.releaseTime ? multisigWallet.releaseTime() : null,
      amount: balance.amount(),
      isMain: false,
    }
)
```

And this selector returns object, not `any[]` (please check line 20)